integration_tests/cases/common/function/time_bucket.result

```diff
@@ -49,12 +49,12 @@ Timestamp(1659283200000),
 SELECT time_bucket(`timestamp`, 'P1W') FROM `02_function_time_bucket_table`;
 
 time_bucket(02_function_time_bucket_table.timestamp,Utf8("P1W")),
-Timestamp(1652918400000),
-Timestamp(1657756800000),
-Timestamp(1657756800000),
-Timestamp(1657756800000),
-Timestamp(1657756800000),
-Timestamp(1657756800000),
+Timestamp(1656259200000),
```

Comment:
I still have question about this week result.
The original ts is `1659577423000`, 2022-07-04 09:43:43+08

When truncate by week, it should be `2022-07-04 00:00:00+08`.

Tested aginst PostgreSQL
```
SELECT
    date_trunc('week', '2022-07-04T09:43:43+08'::timestamp with time zone),
    extract(epoch FROM date_trunc('week', '2022-07-04T09:43:43+08'::timestamp with time zone));
```
Output
```
       date_trunc       |      extract      
------------------------+-------------------
 2022-07-04 00:00:00+08 | 1656864000.000000
(1 row)
```

Reply:
It should be a time zone problem.  I am currently dealing with it after `trunc`. this part should be placed before the function call

Reply:
👍 

Also could you update those sql to something like
```
SELECT timestamp, time_bucket(`timestamp`, 'P1D') FROM `02_function_time_bucket_table` order by timestamp;
```